package.json

```diff
@@ -64,7 +64,7 @@
     "vinyl-source-stream": "~1.1.0"
   },
   "dependencies": {
-    "commander": "^2.11.0",
+    "commander": "~2.11.0",
```

Comment:
FYI, commander 2.12 breaks two of your tests as it changes the way the output looks. It wasn't a trivial thing to fix, so I've pinned it to 2.11, and let you decide how you want to fix them.